test/c/CMakeLists.txt

```diff
@@ -13,8 +13,9 @@ file(GLOB_RECURSE TINYSPLINE_TESTS_SOURCE_FILES "*.c")
 include_directories(cutest)
 add_executable(tinyspline_tests ${TINYSPLINE_TESTS_SOURCE_FILES})
 target_link_libraries(tinyspline_tests
-	LINK_PUBLIC tinyspline_static
-	${TINYSPLINE_C_STATIC_LIBRARIES})
+	PRIVATE tinyspline)
+
+add_test(tinyspline_tests tinyspline_tests)
```

Comment:
What is the purpose of this line?

Reply:
This change + `enable_testing()` in the root of the project allows executing `make test`.
It's the cmake interface to `ctest`.

Reply:
Nice. Thanks.